chef/cookbooks/hyperv/recipes/config_ceilometer.rb

```diff
@@ -0,0 +1,110 @@
+raise if not node[:platform] == 'windows'
```

Comment:
In the past we did not have any error messages here, but we could add them of course.